deploy/apigee/deploy_proxy.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Apigee proxy setup
+# Reference: https://cloud.google.com/apigee/docs/api-platform/get-started/install-cli
+
+set -e
+
+ENV=$1
+
+if [[ $ENV == "" ]]; then
+  echo "Missing arg 1 (env)"
+  exit 1
+fi
+
+PROJECT_ID=$(yq eval '.project' config.yaml)
```

Comment:
I wonder if this script should live under gke where the rest of the one time setup scripts are because:

1. The instructions for running this script is in the README for that folder
1. I think this script needs to be run from that folder in order for this line to work right? because I don't think there is a config.yaml file in this folder?

Reply:
That makes sense to me. Moved to gke directory.